4 warnings ignored

Currently, there are 4 warnings being ignored. These are the following:

```
client.h: In function ‘isValidIpAddress’:
client.h:68:18: warning: implicit declaration of function ‘inet_pton’ [-Wimplicit-function-declaration]
   68 |     int result = inet_pton(AF_INET, ipAddress, &(sa.sin_addr));
      |                  ^~~~~~~~~
```
```
main.c: In function ‘main’:
main.c:69:34: warning: implicit declaration of function ‘inet_addr’ [-Wimplicit-function-declaration]
   69 |  serveraddress.sin_addr.s_addr = inet_addr(server_ip);
      |                                  ^~~~~~~~~
```
```
main.c: In function ‘send_packet’:
main.c:120:45: warning: implicit declaration of function ‘inet_ntop’ [-Wimplicit-function-declaration]
  120 |   printf("Data Received from server %s:\n", inet_ntop(AF_INET,&serveraddr.sin_addr,
      |                                             ^~~~~~~~~
```
```
main.c:120:38: warning: format ‘%s’ expects argument of type ‘char *’, but argument 2 has type ‘int’ [-Wformat ]
  120 |   printf("Data Received from server %s:\n", inet_ntop(AF_INET,&serveraddr.sin_addr,
      |                                     ~^      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                      |      |
      |                                      char * int
      |                                     %d
  121 |     servername,sizeof(servername)));
      |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~    
```

None of these break anything, but they might in the future if they are being ignored.

--- a/client.h
+++ b/client.h
@@ -2,6 +2,8 @@
     Author: Niek van Leeuwen
     Date:   07-11-2019
 */
+
+#include <arpa/inet.h>
 #include <ctype.h>
 #include <netinet/in.h>
 #include <stdbool.h>

